### PR TITLE
CI checks

### PR DESCRIPTION
**Pull request by zimbatm**
Add more CI checks as promised

### CODE REVIEW

File: pkgs/intreehooks/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytoml
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "intreehooks";
```
Comment (adisbladis) [inline pkgs/intreehooks/default.nix]: No need to add `intreehooks` to the overlay, I've backported it to 19.09.

**Comment by zimbatm** [comment]
@adisbladis fixed

File: pkgs/intreehooks/default.nix (hunk 1 above)
Comment (zimbatm) [reply pkgs/intreehooks/default.nix]: maybe it's worth keeping it?

>  attribute 'intreehooks' missing, at /builds/nix-community/poetry2nix/default.nix:209:34
(from CI)

**Comment by zimbatm** [comment]
@adisbladis re-added intreehooks to fix the CI

File: pkgs/intreehooks/default.nix (hunk 1 above)
Comment (zimbatm) [reply pkgs/intreehooks/default.nix]: re-added the commit

**Pull request merged**